Model/Cron.php

```diff
@@ -319,6 +326,11 @@ class Cron
      */
     protected $invoiceHelper;
 
+    /**
+     * @var CaseManagement
+     */
+    protected $caseManagementHelper;
```

Comment:
Why not private?

Reply:
Updated in https://github.com/Adyen/adyen-magento2/pull/1274/commits/74d66b9a058723e7b582048bc510e1c39e2e1e20